ajax_select/fields.py

```diff
@@ -11,8 +13,10 @@
 from django.utils.encoding import force_text
 from django.utils.module_loading import import_string
 from django.utils.safestring import mark_safe
-from django.utils.six import text_type
-from django.utils.translation import ugettext as _
+if sys.version_info.major >= 3:
+    from django.utils.translation import gettext as _
+else:
+    from django.utils.translation import ugettext as _
 
 from ajax_select.registry import registry
```

Comment:
ajax_select/fields.py:21:1: E402 module level import not at top of file

I think you can just move this line it up to line 16. flake just looks for non-import statements (`if`) occurring before import statements.